utils/char_buffer_str.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef UTILS_CHAR_BUFFER_STR_H_
+#define UTILS_CHAR_BUFFER_STR_H_
+
+#include <stdlib.h>
+#include <string.h>
+#include "common/ta_errors.h"
+#include "utils/char_buffer.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Get string from char_buffer_t
+ * @param[in] char_buff char_buffer_t object
+ * @param[out] json_result Response message
+ *
+ * @return
+ * - SC_OK on success
+ * - non-zero on error
+ */
+static inline status_t char_buffer_to_str(char_buffer_t* char_buff, char** json_result) {
```

Comment:
I have no idea why it was named after `char_buffer`.

Reply:
`char_buffer` is named by IF.

Reply:
Entangled called it `char_buffer_t`.

Reply:
Can we rename this file to `char_buffer_ex` as an extension to entangle?

Reply:
so let me rename it to  `char_buffer_t_to_str`

Reply:
How about the scheme such as `xxx_from_char_buffer`? where `xxx`is the use scenario.

Reply:
The suffix `_t` is used for typedef'd remarks. Keep it as `char_buffer`. I just want to avoid misinterpretation.